whois/__init__.py

```diff
@@ -23,7 +23,7 @@
 )
 
 
-def whois(url, command=False, flags=0, executable="whois", inc_raw=False, quiet=False, convert_punycode=True):
+def whois(url, command=False, flags=0, executable="whois", executable_opts=[], inc_raw=False, quiet=False, convert_punycode=True):
```

Comment:
Can you change the default argument to None?

Unfortunately using a mutable default argument in Python leads to confusing results: 
https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument